StaticAnalysisQualityGate/StaticAnalysisQualityService.ts

```diff
@@ -0,0 +1,117 @@
+import * as tl from 'azure-pipelines-task-lib/task';
```

Comment:
Should add copy right comment.

Reply:
The copyright already exists in the code, and it will be merged into when it is all prepared here